pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -379,16 +379,16 @@ public String getInstanceId() {
         getControllerPort() == null ? Arrays.asList("http") : Arrays.asList());
   }
 
-  public String getControllerAccessProtocolProperty(String protocol, String property) {
-    return getProperty(CONTROLLER_ACCESS_PROTOCOLS + "." + protocol + "." + property);
+  public String getControllerAccessProtocolProperty(String name, String property) {
```

Comment:
Is this a backward incompatible change?

Reply:
only java param name changed, not the property. I reverted that change to minimize confusion.